MUON/MUONgeometry/AliMUONVGeometryBuilder.cxx

```diff
@@ -288,7 +290,8 @@ void AliMUONVGeometryBuilder::SetTransformation(Int_t moduleId,
     = TGeoCombiTrans(translation, rotation);
 
   // Apply frame transform
-  TGeoHMatrix newTransform = ConvertTransform(transformation);
+  TGeoHMatrix newTransform;
+  assert(newTransform = ConvertTransform(transformation));
```

Comment:
I guess the assert inclusion is a left over from some tests ?

Reply:
Similar warnings of uninitialised variables, when TGeoMatrix constructors are called, come from other parts of the MUON geometry builder, not only the trigger part. A common strategy should better address those issues, this is kind of rapid fix to understand what the problem is.

Reply:
@bovulpes I have two issues with this line. 

- I don't understand what it is supposed to test ? 
- also, that's dangerous to execute code in an assert, as the assert is disabled in release mode (i.e. when NDEBUG is defined)